kafka-scheduler/src/main/java/com/mesosphere/dcos/kafka/offer/PersistentOfferRequirementProvider.java

```diff
@@ -364,6 +369,7 @@ private static String getBrokerCmd(KafkaSchedulerConfiguration config) {
 
         CommandInfo.Builder cmdBuilder = CommandInfo.newBuilder()
                 .setValue(Joiner.on(" && ").join(Arrays.asList(
+                        config.getExecutorConfiguration().getCommand(),
```

Comment:
FYI this is only for https://dcosjira.atlassian.net/browse/DCOS-429?workflowName=Default+Workflow&stepId=34

`ifconfig lo up` needs to be done before anything else. I had initially broke out the whole command as a config parameter but it looks like you guys want to have it hardcoded in here. Are you okay with having this prepended to the command in this way?